custom_dc/custom_data.md

```diff
@@ -116,11 +167,209 @@ geoId/06,2021,555,666
+      "entityType": "Country",
+      "provenance": "UN_WHO",
+      "observationProperties" : {
+        "unit": "percentage"
+      }
+    }
+  },
+  "variables": {
+    "Adult_curr_cig_smokers": {
+      "name": "Adult Current Cigarette Smokers",
+      "description": "Percentage of smokers in the total adult population",
+      "searchDescriptions": [
+        "Prevalence of smoking among adults in world countries in the years 2016 - 2019."
+      ],
+      "group": "WHO"
+    },
+    "Adult_curr_cig_smokers_female": {
+      "name": "Adult Current Cigarette Smokers Female",
+      "description": "Percentage of smokers in the female adult population",
+      "searchDescriptions": [
+        "Prevalence of smoking among adult women in world countries in the years 2016 - 2019."
+      ],
+      "group": "WHO"
+    },
+      "Adult_curr_cig_smokers_male": {
+      "name": "Adult Current Cigarette Smokers Male",
+      "description": "Percentage of smokers in the male adult population",
+      "searchDescriptions": [
+        "Prevalence of smoking among adult men in world countries in the years 2016 - 2019."
+      ],
+      "group": "WHO"
+    }
+  },
+ "sources": {
+    "custom.who.int": {
+      "url": "https://custom.who.int",
+      "provenances": {
+        "UN_WHO": "https://custom.who.int/data/gho/indicator-metadata-registry/imr-details/6128"
+      }
+    }
+  }
+}
+```
+The following fields are specific to the variable-per-column format:
+
+- `input_files`:
+  - `entityType`: This must be an existing entity class in the Data Commons knowledge graph; it's most commonly a [place type](/place_types.html).
+- `variables`: This section is optional but recommended. You can use it to override names and associate additional properties with the statistical variables in the files, using the parameters described below. All parameters are optional.
+  - `name`: A human-friendly readable name that will be shown throughout the UI.
+  - `description`: A more detailed name that will be shown in the Statistical Variable Explorer.
+  - `searchDescriptions`: This is a comma-separated list of natural-language text descriptions of the variable; these descriptions will be used to generate embeddings for the NL query interface.
+  - `group`: This will display the variables as a group in the Statistical Variable Explorer, using the name you provide as heading. 
+
+The other fields are explained in the [Data config file specification reference](#json-ref).
+
+## Prepare your data using explicit schema
+
+In this section, we will walk you through a concrete example of how to go about setting up your CSV, MCF and JSON files.
+
+### Write the MCF file {#mcf}
+
+Nodes in the Data Commons knowledge graph are defined in Metadata Content Format (MCF). For custom Data Commons using explicit schema, you must define your statistical variables using MCF. Here's an example of defining the same statistical variables in the WHO data in MCF:
+
```

Comment:
Should we add a note on where the MCF file should be kept?

It didn't seem to work for on.org when placed in the same folder as csv files.

Reply:
So, they are probably configuring something wrong. Like did they include format: variablePerRow in their JSON? Because having everything in one file is how UN and USAFacts are doing it, and how I do it locally, so it definitely works and most likely is another configuration error on their side.

Reply:
Sorry, I realized I misspoke. The MCF file must be in the same directory as the config.json. it doesn't have to be with the CSV files. Anyway, good point to mention this; I'll add it.

Reply:
I added a diagram at the top of the page, that will hopefully make this clearer for readers.